lib/image_db.rb

```diff
@@ -85,8 +95,8 @@ def full_update
 
     (1..page_count).each do |page|
       data = ImageDB.dribbble_client.get_user(dribbble_user).likes page: page
-      p ({ :player => dribbble_user, :page => page })
-      data.each {|l| @images.add l.html_url }
+      p ({ player: dribbble_user, page: page })
```

Comment:
`(...)` interpreted as grouped expression.<br>Do not write to stdout. Use Rails' logger if you want to log.